pipeline_dp/dataset_histograms/computing_histograms.py

```diff
@@ -99,8 +110,9 @@ def _compute_frequency_histogram_helper(
     """
     col = backend.map_tuple(
         col, lambda n, f:
-        (_to_bin_lower(n),
-         hist.FrequencyBin(lower=_to_bin_lower(n), count=f, sum=f * n, max=n)),
+        (_to_bin_lower_logarithmic(n),
```

Comment:
should we calculate it only once?

Reply:
I meant that in the next line we call again _to_bin_lower(n) with the same argument. We can cache it or just store in variable. The second option is better because we know that n will be always different.

Reply:
ah, good point! Yeah, let's compute it only once. Probably it would more readable to extract lambda in an internal function.